docs/TURN.md

```diff
@@ -39,60 +62,79 @@ The configuration of Nextcloud Talk mainly depends on your desired usage:
          stale-nonce
          no-loopback-peers # Only on coTURN below v4.5.1.0!
          no-multicast-peers
-   - <details><summary>(D)TLS is currently not supported by Nextcloud Talk and does not have any real security benefit anyway. Click here for more details.</summary>
-
-     - See the following discussions why **(D)TLS** for TURN has no real security benefit and that Nextcloud Talk not supporting it: https://github.com/coturn/coturn/issues/33, https://github.com/nextcloud/spreed/issues/257
-     - However for completeness: When using (D)TLS, you need to provide the path to your certificate and key files, and it is highly recommended to adjust the cipher list:
-
-           tls-listening-port=<yourChosenPortNumber>
-           fingerprint
-           lt-cred-mech # Only on coTURN below v4.5.0.8!
-           use-auth-secret
-           static-auth-secret=<yourChosen/GeneratedSecret>
-           realm=your.domain.org
-           total-quota=100
-           bps-capacity=0
-           stale-nonce
-           cert=/path/to/your/cert.pem
-           pkey=/path/to/your/privkey.pem
-           cipher-list="ECDH+AESGCM:DH+AESGCM:ECDH+AES256:DH+AES256:ECDH+AES128:DH+AES:ECDH+3DES:DH+3DES:RSA+AES:RSA+3DES:!ADH:!AECDH:!MD5"
-           no-loopback-peers
-           no-multicast-peers # Only on coTURN below v4.5.1.0!
-
-     - Note that `listening-port`, `alt-listening-port`, `tls-listening-port` and `alt-tls-listening-port` can all be used for (D)TLS and plain text connections. It depends on the client request protocol only, TURN vs _TURNS_ (TURN over TLS). Hence there is usually no point to setup more then one port. Also Nextcloud Talk can only be configured to use a single port.
-     - A working cipher example is provided above, that is also used within most other guides. But it makes totally sense to **use the cipher-list from your Nextcloud webserver** to have the same compatibility versus security versus performance for both.
-     - If you want it damn secure, you can also configure a custom [Diffie-Hellman](https://en.wikipedia.org/wiki/Diffie–Hellman_key_exchange) file and/or disable TLSv1.0 + TLSv1.1. But again, it does not make much sense to handle it different here than for the webserver. Just decide how much compatibility you need and security/performance you want and configure webserver + coTURN the same:
+
+
+#### 3.1  (D)TLS configuration
```

Comment:
Is it a good idea to have this part fully expended? As said, I left it inside for completeness only and since it was there before I found it being not supported anyway. I fear that with this, many readers will setup TLS on their TURN server which will then be just an unused overhead.

We could leave the hint, that it is not supported + links, so readers will not ask, when recognising that coTURN has (D)TLS support.
As fast as support is there, actually it does not sound that complicated from the linked issues, we could re-add this part in a proper way, that targets the actual use case: When firewalls do not allow unencrypted traffic on signalling layer (WebRTC layer is encrypted anyway).
Also the cipher requires an update. On all guides around you'll find the exact same, which is based on the example from spreedbox, but actually this should be updated to match current standards.

Reply:
The details html tag in docs prevents all markdown inside to be rendered... Either we remove it or it has to be displayed

Reply:
@nickvergessen 
Ah it's html (I guess the `<>` is a hint for that), that explains also that it is parsed different on different markdown viewers (Nextcloud forum vs GitHub vs others).

Then I vote for removing at least the config details and show support hint + links only. Docs should not contain something that cannot be used together with the software, IMO.